### ISSUE TITLE
Link component not updating html tag attributes

### ISSUE THREAD

**Issue by pushparajbhatta** [inertiajs/inertia#1341]
### Versions:

- `@inertiajs/inertia` version: 0.11.0
- `@inertiajs/inertia-vue3` version: 0.6.0

### Describe the problem:

As per Inertia JS docs:
Link component:
`This is a light wrapper around a standard anchor <a> link that intercepts click events and prevents full page reloads from occurring.`

As stated, it will definitely offer faster UI experiences. But in some cases like when the language translations depend on the "lang" attribute of <html lang="en"> tag, then it creates issues. Is there a way to override these behaviors of Link components ? I tried the Replace option, but it did not work. I used normal anchor tag to resolve the issue. Can Link be improved to support such use cases (i.e. where it requires a full page reload)?

A lot of users are experiencing this issue while using it for language translations.
Example: https://github.com/xiCO2k/laravel-vue-i18n/issues/62

### Steps to reproduce:

1. Use **Link** component
2. Create an HTML tag with test attribute <html test="{{time()}}">
4. Load the page in browser
5. Inspect and notice the value of the "test" attribute
6. Now click the link generated from the Link component
7. The value of the "test" attribute will remain the same, and only after a manual refresh with F5, it will show the updated value

**Comment by reinink** [comment]
Hey! Thanks so much for your interest in Inertia.js and for sharing this issue/suggestion.

In an attempt to get on top of the issues and pull requests on this project I am going through all the older issues and PRs and closing them, as there's a decent chance that they have since been resolved or are simply not relevant any longer. My hope is that with a "clean slate" me and the other project maintainers will be able to better keep on top of issues and PRs moving forward.

Of course there's a chance that this issue is still relevant, and if that's the case feel free to simply submit a new issue. The only thing I ask is that you please include a super minimal reproduction of the issue as a Git repo. This makes it much easier for us to reproduce things on our end and ultimately fix it.

Really not trying to be dismissive here, I just need to find a way to get this project back into a state that I am able to maintain it. Hope that makes sense! ❤️
